browser/main/lib/dataApi/createNoteFromUrl.js

```diff
@@ -0,0 +1,94 @@
+const http = require('http')
+const https = require('https')
+const TurndownService = require('turndown')
+const createNote = require('./createNote')
+
+import { hashHistory } from 'react-router'
+import ee from 'browser/main/lib/eventEmitter'
+
+function validateUrl(str) {
```

Comment:
This method could be simplyfied to 
```
return str.matches(new RegExp('^(?:.......'));
```

Reply:
Oh, my idea would have introduced the opportunity for null pointers.. So it's good that you ignored my concern :D